### DISCUSSION TITLE
Set map bounds

### DISCUSSION THREAD

**Discussion by marcinkieruzel** [giorgiabosello/google-maps-react-markers#29] [General]
Hi, 
the documentations indicates function "setMapBounds({ ...mapBounds, bounds: [sw.lng(), sw.lat(), ne.lng(), ne.lat()], zoom })" 
Where does it come from? Is it any custom function or it comes from the library itself?

**Comment by giorgiabosello** [comment]
It's simply an internal state. The example is to suggest how to use bounds with the useSupercluster hook.

**Reply by marcinkieruzel** [reply]
Hi, thanks a lot – its obvious. 
Thanks a lot
